en/docs/references/create-a-record-using-statement-editor

```diff
@@ -0,0 +1,35 @@
+# Construct a record using the Statement-Editor
```

Comment:
```suggestion
# Construct a record using the Statement Editor
```
This cannot be a level-1 heading. Please check and fix.

Reply:
This is a level 1 heading as it is the main title of the page.